### ISSUE TITLE
Error while trying to install NAO on NVDA 2024.1

### ISSUE THREAD

**Issue by aamir51** [sharkboyto/nao#47]
Hi
I get this error While trying to install NAO on NVDA 2024.1: 

> [Window Title]
> Error
> 
> [Content]
> Failed to install add-on from D:\Downloads\nao-2024.1.0.nvda-addon
> 
> [OK]
>
Log below: 

> DEBUG - addonHandler.Addon.loadModule (14:13:15.862) - systemUtils.ExecAndPump(<function installAddonBundle at 0x031681B8>) (14792):
> Importing module installTasks from plugin Addon ('nao', running=False)
> ERROR - addonHandler.installAddonBundle (14:13:15.862) - systemUtils.ExecAndPump(<function installAddonBundle at 0x031681B8>) (14792):
> task 'onInstall' on addon 'nao' failed
> Traceback (most recent call last):
>   File "addonHandler\__init__.pyc", line 434, in installAddonBundle
>   File "addonHandler\__init__.pyc", line 717, in runInstallTask
>   File "C:\Users\Username-PC\AppData\Roaming\nvda\addons\nao\installTasks.py", line 44, in onInstall
>   File "C:\Users\Username-PC\AppData\Roaming\nvda\addons\nao\installTasks.py", line 35, in request
>   File "addonHandler\__init__.pyc", line 784, in getCodeAddon
> addonHandler.AddonError: Code does not belong to an addon
> DEBUG - addonHandler.Addon.completeRemove (14:13:15.874) - systemUtils.ExecAndPump(<function installAddonBundle at 0x031681B8>) (14792):
> removing addon nao from the list of disabled / blocked add-ons
> DEBUGWARNING - systemUtils.ExecAndPump.run (14:13:15.874) - systemUtils.ExecAndPump(<function installAddonBundle at 0x031681B8>) (14792):
> task had errors
> Traceback (most recent call last):
>   File "addonHandler\__init__.pyc", line 434, in installAddonBundle
>   File "addonHandler\__init__.pyc", line 717, in runInstallTask
>   File "C:\Users\Username-PC\AppData\Roaming\nvda\addons\nao\installTasks.py", line 44, in onInstall
>   File "C:\Users\Username-PC\AppData\Roaming\nvda\addons\nao\installTasks.py", line 35, in request
>   File "addonHandler\__init__.pyc", line 784, in getCodeAddon
> addonHandler.AddonError: Code does not belong to an addon
> 
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File "systemUtils.pyc", line 236, in run
>   File "addonHandler\__init__.pyc", line 439, in installAddonBundle
> addonHandler.AddonError: Installation failed
> IO - speech.speech.speak (14:13:15.964) - MainThread (14536):
> Speaking [LangChangeCommand ('en'), 'Installing Add-on', 'dialog', 'Please wait while the add-on is being installed.', CancellableSpeech (still valid)]
> IO - tones.beep (14:13:16.000) - MainThread (14536):
> Beep at pitch 1760, for 40 ms, left volume 50, right volume 50
> ERROR - gui.addonGui.installAddon (14:13:16.002) - MainThread (14536):
> Error installing  addon bundle from D:\Downloads\nao-2024.1.0.nvda-addon
> Traceback (most recent call last):
>   File "addonHandler\__init__.pyc", line 434, in installAddonBundle
>   File "addonHandler\__init__.pyc", line 717, in runInstallTask
>   File "C:\Users\Username-PC\AppData\Roaming\nvda\addons\nao\installTasks.py", line 44, in onInstall
>   File "C:\Users\Username-PC\AppData\Roaming\nvda\addons\nao\installTasks.py", line 35, in request
>   File "addonHandler\__init__.pyc", line 784, in getCodeAddon
> addonHandler.AddonError: Code does not belong to an addon
> 
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File "gui\addonGui.pyc", line 222, in installAddon
>   File "systemUtils.pyc", line 232, in __init__
>   File "systemUtils.pyc", line 236, in run
>   File "addonHandler\__init__.pyc", line 439, in installAddonBundle
> addonHandler.AddonError: Installation failed
> IO - speech.speech.speak (14:13:16.036) - MainThread (14536):
> Speaking [LangChangeCommand ('en'), 'Error', 'dialog', 'Failed to install add-on from D:\\Downloads\\nao-2024.1.0.nvda-addon', CancellableSpeech (still valid)]
> IO - speech.speech.speak (14:13:16.038) - MainThread (14536):
> Speaking [LangChangeCommand ('en'), 'OK', 'button', CancellableSpeech (still valid)]
> IO - inputCore.InputManager.executeGesture (14:13:18.135) - winInputHook (10024):
> Input: kb(desktop):enter
> IO - speech.speech.speak (14:13:18.182) - MainThread (14536):
> Speaking [LangChangeCommand ('en'), 'Installing Add-on', 'dialog', 'Please wait while the add-on is being installed.', CancellableSpeech (still valid)]
> IO - speech.speech.speak (14:13:18.194) - MainThread (14536):
> Speaking [LangChangeCommand ('en'), 'Installing Add-on', CancellableSpeech (still valid)]
> 
> 
@sharkboyto

**Comment by dreinn** [comment]
sorry for the late answer.
I think you have a corrupted package, since we have no problem installing Nao.
I suggest you to download it again.
